### PR TITLE
Create new landing

### PR DESCRIPTION
**Pull request by marharyta**
Recreate landing for HelsinkiJS  completly.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,67 +1,56 @@
...
+  <link rel="mask-icon" href="./safari-pinned-tab.svg" color="#5bbad5">
+  <meta name="msapplication-TileColor" content="#ffc40d">
+  <meta name="theme-color" content="#ffffff">
+</head>
+
+<body>
+  <div class="wrapper">
+    <div class="placeholder">
+      <div class="image">
+        <img src="https://res.cloudinary.com/toughbyte/image/upload/b_transparent,c_pad,g_center,h_230,w_230/v1448022416/bulcgwowlfcdnpdqv4yi.jpg"
```
[the remark this answers is not in the document]
Comment (marharyta) [reply index.html]: Well, if you can provide me with the assets, I can do that.

[the remark this answers is not in the document]
Comment (eemeli) [reply index.html]: Dropping URL parts from the source leaves me with this, which is at least slightly better than the current one (though still jpg, which is wrong):

https://res.cloudinary.com/toughbyte/image/upload/bulcgwowlfcdnpdqv4yi.jpg

[the remark this answers is not in the document]
Comment (sarukuku) [reply index.html]: I have the original assets I think. Let me see.

[the remark this answers is not in the document]
Comment (sarukuku) [reply index.html]: Here's a link to download an svg version of the logo: https://www.dropbox.com/s/ouv12nmzazy2i5v/HelsinkJS_logo.svg.zip?dl=1

**Comment by marharyta** [comment]
screen shot 2018-12-05 at 16 44 28
I updated background color: now looks hipster af @eemeli @sarukuku

**Comment by marharyta** [comment]
@eemeli @sarukuku @olegp So, any other action points?

**Comment by eemeli** [comment]
I think with the all-yellow look the logo needs a bit more space around it; it's lost its square frame, and its components are now interacting a bit too much with the title. Or maybe a square black frame? Something, at least.

I do really like the overall black-on-yellow hipster look, but the logo & title don't really mesh well together. On the other hand, we could go live with this and ask for PRs from our community?

**Comment by marharyta** [comment]
@eemeli Agree! It is nice to have a look with fresh eyes!

**Pull request merged**